src/components/forms/PlanForm/index.tsx

```diff
@@ -342,6 +363,15 @@ const PlanForm = (props: PlanFormProps) => {
                 {isPlanTypeEnabled(InterventionType.MDAPoint) && (
                   <option value={InterventionType.MDAPoint}>{MDA_POINT_TITLE}</option>
                 )}
+                {isPlanTypeEnabled(InterventionType.DynamicFI) && (
```

Comment:
possible as map?

Reply:
Have not yet changed this to a map.  Trickier than the others to make work.